packages/fxa-auth-server/lib/payments/configuration/base.ts

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import joi from '@hapi/joi';
+
+export type CapabilityConfig = {
+  [key: string]: string;
+};
+
+const capabilitySchema = joi
+  .object()
+  .pattern(joi.string(), joi.string().min(1))
+  .min(1);
+
+export type UrlConfig = {
+  download?: string;
+  webIcon?: string;
+  emailIcon?: string;
+  termsOfService?: string;
+  termsOfServiceDownload?: string;
+  privacyNotice?: string;
+  privacyNoticeDownload?: string;
+  playStore?: string;
+  appStore?: string;
+};
+
+const urlsSchema = joi.object({
+  download: joi.string().uri(),
+  webIcon: joi.string().uri(),
+  emailIcon: joi.string().uri(),
+  termsOfService: joi.string().uri(),
+  termsOfServiceDownload: joi.string().uri(),
+  privacyNotice: joi.string().uri(),
+  privacyNoticeDownload: joi.string().uri(),
+  playStore: joi.string().uri(),
+  appStore: joi.string().uri(),
+});
+
+export type UiContentConfig = {
+  subtitle?: string;
+  details?: string[];
+  successActionButtonLabel?: string;
+  upgradeCTA?: string;
+};
+
+const uiContentSchema = joi.object({
+  subtitle: joi.string(),
+  details: joi.array().items(joi.string()),
+  successActionButtonLabel: joi.string(),
+  upgradeCTA: joi.string(),
+});
+
+export type StyleConfig = {
+  webIconBackground?: string;
```

Comment:
Huh; I see this used in the payments front-end, but it's not in our [existing Joi product metadata validator](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/routes/validators.js#L433-L500), even as an optional key.

Interestingly, that seems to be the case for a number of the optional metadata keys.